### PR TITLE
adds make go command

### PR DESCRIPTION
**Pull request by ghepting**
`make go`

### CODE REVIEW

**Review by ryunsong-contentful — APPROVED** [review]

File: apps/google-analytics-4/lambda/Makefile [hunk 1]
```diff
@@ -5,6 +5,10 @@ all: help
 start-app:
 	@docker compose up --build
 
+## make go: Starts a docker image for dynamodb, our serverless lambda, and frontend for local development
```
Comment (ryunsong-contentful) [inline apps/google-analytics-4/lambda/Makefile:8]: should we remove start-app then? What's the difference?

Comment (ghepting) [reply apps/google-analytics-4/lambda/Makefile:8]: i wanted to make it backward compatible, but i'd like to :)

**Review by matthew-gordon — APPROVED** [review]

**Comment by matthew-gordon** [comment]
:sob: I was hoping this day would come :fist:

File: apps/google-analytics-4/lambda/Makefile (hunk 1 above)
Comment (matthew-gordon) [reply apps/google-analytics-4/lambda/Makefile:8]: 💣 💥 Nuke it!

Comment (ghepting) [reply apps/google-analytics-4/lambda/Makefile:4]: removed `start-app`

**Pull request merged**